### ISSUE TITLE
[Bug]: Youtube video preview not playing continuously and gets interrupted

### ISSUE THREAD

**Issue by Dean-Corso** [Floorp-Projects/Floorp#1523]
### Pre-Submission Checklist

- [ ] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [ ] I have checked with a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [ ] I have included relevant screenshots or console outputs.
- [ ] I have checked that this problem is not occurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

I found this browser today and gave it a try. After installing & setup I visited Youtube website and did enter some search term. Now you got a list full of videos which starting to play if you hover with the mouse pointer on the them and right here the problem starts. If you hover with the mouse on any video then it starts playing for a moment and then it stops and starts again. Also same issue happens when you move the mouse over the video area during its running. The playback always stops & runs again. Normally this does not happen in any browser like Firefox / Chrome etc. Please check that and fix it because its super annoying.

### Reproduction Steps

1. Go to [youtube](https://www.youtube.com/)
2. Enter any term to search for like Tennis
3. Now you get many suggestions of videos
4. Hover with the mouse over any of them
5. Click on mute / unmute button to hear audio / subtitle etc
6. Just hover with mouse over the video area a little
7. Now you will notice that playing / stop / playing / stop problem
8. This problem never happened when using Firefox / Chrome etc

### Expected Behavior

The video playback on preview should play the video continuously without any interruption of the playback so long I keep the mouse pointer into the video area field.

### Actual Behavior

The video playback on preview gets interrupted.

### Screenshots

No Screenshot!

### Browser Version

11.21.0 (64-Bit)

### Operating System

Windows 10 x64

**Comment by Dean-Corso** [comment]
Hi again,

I made a little video now so that you can see the problem I mean when trying to preview YT videos. Just watch and try!

https://github.com/user-attachments/assets/de73e2be-139d-496e-8174-c25e7d638d44

**Comment by typeling1578** [comment]
I still cannot reproduce the issue.

**Comment by Dean-Corso** [comment]
But you can see it that there is an issue. Some kind of flickering / restart video playback each time when I hover with the mouse over it. I don't have this problem with any other browser's like Firefox / ZEN itself only with Floorp.

Info: I found something, when I open / show the side panel of Bookmarks on the left side and hove with mouse over any video preview on YT (as you can see in my video) then the video does not flash / flicker / restarts etc. When I hide the side panel / bookmark again then the video preview does flash again like in the video I did post yesterday. Somewhere should be any bug.

**Comment by creeper-0910** [comment]
Hmmm... I cannot reproduce this issue.
Are you using any extensions?

**Comment by Dean-Corso** [comment]
Yes, I tried same without AddOns and also with delete cache etc. Always get the same problem to see more or less. So in my video you can also see that the icon is showing in the URL bar when I hover with the mouse over any video preview but the icon mostly does flash twice before it stays so long I have the mouse over the video preview. Now, when I do same with normal Firefox then the icon is not flashing twice and its just showing (mouse over video preview) and stays. Why does Floorp flash? 

Maybe you changed anything about that in source you should better not change maybe some notification issue. Did you add some more API functions / custom handling there to update the window handle or something? Anyway, so there is a issue with Floorp I do not have in Firefox / ZEN / Nightly etc only in Floorp. Do you need some more system info's or something? I'm running on Win10 x64. I hope you can find that problem and fix it because its annoying somehow.

**Comment by typeling1578** [comment]
However, we are unable to reproduce this issue.
I am not a dedicated member of the Floorp team, so I am not fully aware of the details, but as far as I know, no modifications have been made to the rendering engine in Floorp.

**Comment by Dean-Corso** [comment]
Hi, so I did reset Floorp browser today using the reset function and after that the problem was gone. That's the good news. Bad news is I don't know why and what the real problem was. I did same steps as before to make same settings / installing extensions but all is fine so far. Only problem I see is that Floorp has a problem to handle installed themes, In adjust window where you can put controls into browser or remove them etc its showing somehow wrong also there is some color problem (status bar has dark color overlay)....
![FL_2024-12-17_231414](https://github.com/user-attachments/assets/09167980-a5cd-42b2-99e0-5ac87f9d8b31)
...if I deactivate the design then all looks normal.
